package.json

```diff
@@ -41,7 +41,7 @@
     "prettier": "^1.16.4",
     "require-npm4-to-publish": "^1.0.0",
     "resin-lint": "^2.0.1",
-    "tar-utils": "^1.1.0",
+    "tar-utils": "^2.0.0-fix-normalize-path-separator-5e0d2fd21e1eaeaf1aab04ec223ff37aaa90880b",
```

Comment:
I'll update this line before merging this PR, when the tar-utils PR is merged:
https://github.com/balena-io-modules/node-tar-utils/pull/8/files